### ISSUE TITLE
Make a website

### ISSUE THREAD

**Issue by chadwhitacre** [opensourcepledge/osspledge.com#6]
Depends on brand identity (#8).

**Comment by chadwhitacre** [comment]
First PR is in https://github.com/osspledge/osspledge.com/pull/1. Sentry will be the first company with a profile, Astral is on deck to be the second. We have a couple more ready after that. Let's see how far we can get in a week! 💃

**Comment by vladh** [comment]
This continues in #27.

**Comment by vladh** [comment]
With #29, there seem to be two big things left before we can onboard members:

* Is the schema final? Can we think of any possible issues we want to head off? The schema should be final before we ask members to use it.
* Does member data display on the website in the way that we want it? This will require another PR.

**Comment by chadwhitacre** [comment]
Let's write up a little howto for the README, we have a few companies ready to alpha test with us, that can help us determine if the schema is final.

The member profile hardcodes a few things right now, most notably dollars per dev. It needs at least a PR to compute that (and whatever else, maybe that's it) from the data. There will be plenty more as we bring the design along.

**Comment by vladh** [comment]
Got it, tracking in #30 and #31.

**Comment by vladh** [comment]
Before launching the initial version of the website, two changes I can think of are:

- [ ] Add members to homepage
- [ ] Improve design of member page

I'm happy to do these, but is there anything else outstanding? 🤔

**Comment by vladh** [comment]
I noticed the green colour in the logo is different from the green colour on the website. Should one be changed to the other?

![2024-08-05-14-18-03](https://github.com/user-attachments/assets/50d50296-dc1e-4965-94da-50c9a7225aa8)

**Comment by chadwhitacre** [comment]
Yes, @stevenplewis (Sentry designer) has a design update completed locally, I'll work with him to make a PR.

![localhost_4321_](https://github.com/user-attachments/assets/72bf12f8-aae9-4f93-9053-ea6283ff396d)

**Comment by chadwhitacre** [comment]
Early adopters to test:

- [ ] Sentry
- [ ] Astral
- [ ] HTTPToolkit
- [ ] Antithesis
- [ ] BrowserBase

**Comment by vladh** [comment]
Talked to @chadwhitacre today, we agreed that if @stevenplewis designs more content for the website as e.g. Figma files, I'm happy to take those files and handle the actual implementation.

**Comment by chadwhitacre** [comment]
We're expecting some initial design mocks this week.

**Comment by vladh** [comment]
Loving this overall, and we can for sure do something fun with the splotches.

I do have some suggestions, but I would say:

1. Let's get a full export of a final draft for this
2. I can handle the implementation ASAP
3. We can refine the details using PRs

**Comment by chadwhitacre** [comment]
Here is the hero image. 💃  🎉 Blobs in progress ...

![hero](https://github.com/user-attachments/assets/adb7a9cf-3fea-4968-bd0a-df37abf8f5fb)

**Comment by chadwhitacre** [comment]
Blobs!

[oss-blobs.zip](https://github.com/user-attachments/files/16805952/oss-blobs.zip)

@vladh We are headed into a 4-day weekend here in the States. I will try to keep an eye on things here in case you're able to jump in on site construction while we're mostly afk. With any luck we will be able to have the site in much better shape a week from now. Huzzah! 🎉 🙌

**Comment by vladh** [comment]
@chadwhitacre It shall be done! 😛

**Comment by chadwhitacre** [comment]
food for thought :o) https://www.instagram.com/reel/C_L9_BLSAT6/

**Comment by vladh** [comment]
@chadwhitacre 

https://github.com/user-attachments/assets/ae3d8c1c-87e9-48d0-826f-30043e59a792

**Comment by chadwhitacre** [comment]
We're off and running with the new site design! 💃 🎉 👏 @stevenplewis @vladh
